syntax = "proto3";

package distance;

service WordsHelper {
    rpc GetDistance (Words) returns (Distance);
    rpc GetFibonacci (FibonacciRequest) returns (stream FibonacciNumber);
}

message FibonacciRequest {
}

message Words {
    string word1 = 1;
    string word2 = 2;
}

message Distance {
    int64 distance = 1;
}

message FibonacciNumber {
    int64 number = 1;
}
